tests/lib/rules/id-blacklist.js

```diff
@@ -297,6 +305,26 @@ ruleTester.run("id-blacklist", rule, {
                     type: "Identifier"
                 }
             ]
+        },
```

Comment:
I'd like to see some more tests for the new option. If we go with the "new style" option schema as suggested [here](https://github.com/eslint/eslint/pull/6314/files#r66665469), it would be great to have tests that make sure the old schema still works and does not treat the strings are regex (i.e. `id-blacklist: [2, 'foo']` would not flag an identifier `foobar` but it would still flag an identifier `foo`)